src/main/java/org/codehaus/mojo/versions/DisplayPropertyUpdatesMojo.java

```diff
@@ -85,6 +85,30 @@
      */
     private Boolean autoLinkItems;
 
+    /**
+     * Whether to allow the major version number to be changed.
+     *
+     * @parameter property="allowMajorUpdates" default-value="true"
+     * @since 1.2
+     */
```

Comment:
If this an addition than you should correct the @since and give the correct version number (2.3 ?)